addons/account/models/account_move.py

```diff
@@ -2837,22 +2899,24 @@ def _check_off_balance(self):
                 if line.reconciled:
                     raise UserError(_('Lines from "Off-Balance Sheet" accounts cannot be reconciled'))
 
-    def _update_check(self):
-        """ Raise Warning to cause rollback if the move is posted, some entries are reconciled or the move is older than the lock date"""
-        move_ids = set()
+    def _affect_tax_report(self):
+        self.ensure_one()
+        return self.tax_ids or self.tax_line_id or self.tag_ids.filtered(lambda x: x.applicability == "taxes")
+
+    def _check_tax_lock_date(self):
```

Comment:
this method should be removed entirely, as we don't want to raise an error anymore

Reply:
maybe not actually

Reply:
You need it. For example if you try to unlink / reset to draft the move